dojo/tools/bugcrowd/parser.py

```diff
@@ -59,6 +60,9 @@ def get_findings(self, filename, test):
             finding.references = References
             finding.severity = self.convert_severity(row.get('priority', 0))
 
+            if row.get('submitted_at'):
+                finding.date = datetime.strptime(row.get('submitted_at'), '%Y-%m-%d %H:%M:%S %Z')
+
```

Comment:
Could you switch to `dateutil` module instead?

Reply:
Done @damiencarol